5-AccessControl/1-call-api-roles/AppCreationScripts/Configure.ps1

```diff
@@ -1,23 +1,18 @@
+
 [CmdletBinding()]
```

Comment:
>CmdletBinding [](http://example.com/codeflow?start=1&length=13)

Upgrade CreateUsersAndAssignRoles.ps1 to use MS Graph PS
Generate this file automatically from code gen if there are one or more App roles "for users"
Do not hard code passwords, rather generate a random string and use it, then print password on console